sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/src/Customization/NodeInput.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Media.VideoAnalyzer.Edge.Models
+{
+    /// <summary>
+    /// Extension for <see cref="NodeInput"/>.
+    /// </summary>
+    public partial class NodeInput
```

Comment:
This name is probably going to be too vague / generic for our Architect Board review, nor do we typically add "Input" to classes where it's clearly used as input. We use no suffix, or either Parameters or Options (usually Options) when bundling a lot of input properties together for a REST call. Could you qualify the "Node" here, like AnalysisNode or something?